test/controllers/v1/supported_ssgs_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module V1
+  class SupportedSsgsControllerTest < ActionDispatch::IntegrationTest
+    setup do
+      SupportedSsgsController.any_instance.stubs(:authenticate_user)
+    end
+
+    context 'index' do
+      should 'lists all supported SSGs' do
+        get v1_supported_ssgs_url
+
+        assert_response :success
```

Comment:
Nit: Maybe also check a record matches the yaml?

Reply:
That would require to copy the algorithm over :slightly_smiling_face: . The spec also checks the schema.